### ISSUE TITLE
Inconsistent naming in istio.io/api/mixer/v1/config/client

### ISSUE THREAD

**Issue by metacosm** [istio/issues#156]
**Did you review istio.io/help and existing issues to identify if this is already solved or being worked on?**:
Yes.

**Bug:**
Possibly :)

**What Version of Istio and Kubernetes are you using, where did you get Istio from, Installation details**
From https://github.com/istio/api/blob/master/mixer/v1/config/client/api_spec.pb.go, both HTTPAPISpec (and consort) and HttpClientConfig are defined. Is there a reason for the inconsistent naming? If not, naming should be harmonized.

**Comment by kyessenov** [comment]
Thanks for catching that.
I suggest we follow https://github.com/envoyproxy/data-plane-api/blob/master/STYLE.md and rename `HTTPAPISpec` to `HttpApiSpec`.

**Comment by metacosm** [comment]
Note that this isn't the only case of inconsistent naming. There are several such occurrences.

**Comment by kyessenov** [comment]
Yes, we're revising APIs. This particular group is in alpha stage, so clearly needs some work to become consistent with the rest of istio.
